cmake/projects/OpenSSL/schemes/url_sha1_openssl_windows.cmake.in

```diff
@@ -66,6 +66,10 @@ else()
   set(do_ms do_ms.bat)
 endif()
 
+if(NOT ASM_SUPPORT)
+  list(APPEND opt " no-asm")
```

Comment:
Please add it to `url_sha1_openssl_windows_1_1_plus.cmake.in` too:
* https://github.com/ruslo/hunter/blob/v0.23.49/cmake/projects/OpenSSL/schemes/url_sha1_openssl_windows_1_1_plus.cmake.in#L102